chains/starknet/addresses.yaml

```diff
@@ -0,0 +1,17 @@
+defaultIsm: "0x0493aaa0f86386541c7ffe0b78e1c929a46fb01ca71021e5327d0576309d3777"
+domainRoutingIsmFactory: "0x0000000000000000000000000000000000000000"
+interchainAccountIsm: "0x0000000000000000000000000000000000000000"
+interchainAccountRouter: "0x0000000000000000000000000000000000000000"
```

Comment:
also want to add
```
interchainGasPaymaster: "0x0000000000000000000000000000000000000000000000000000000000000000"
```

and update the other 0 addresses to `0x0000000000000000000000000000000000000000000000000000000000000000`